src/types.ts

```diff
@@ -196,3 +197,20 @@ export interface Issuer {
   signer: Signer
   alg?: string
 }
+
+/**
+ * Represents the Verification Options that can be passed to the verifyPresentation Method.
+ * The verification will fail if given options are NOT satisfied.
+ */
+export interface VerifyPresentationOptions {
+  domain?: string
+  challenge?: string
+}
+
+/**
+ * Represents the Creation Options that can be passed to the createVerifiablePresentationJwt Method.
+ */
+export interface CreatePresentationOptions {
+  domain?: string
```

Comment:
The `domain` param gets mapped to the `aud` which can also be an array.
I think it would be best if the domain were an optional array as well

Reply:
That would be a different propose than in the issue. Happy to change that, but we should change the requirements there. Also pinging @awoie for that.